src/Service/CommentCRUDService.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Service;
+
+use App\Repository\CommentRepository;
+use App\Entity\Comment;
+
+class CommentCRUDService 
```

Comment:
Are you expecting to split the Comment domain into subparts? We did this for Dog as we have the image, CRUD, and potentially groups plus more. Comments are really just CRUD focused, so don't need to be split out per se.

LOGAF = low

Reply:
Yeah I had a thought on that whilst I was refactoring the functionality inside of Post Repository.